generators/client/templates/angular/src/main/webapp/app/core/auth/user-route-access-service.ts.ejs

```diff
@@ -64,20 +64,16 @@ export class UserRouteAccessService implements CanActivate {
                 if (isDevMode()) {
                     console.error('User has not any of required authorities: ', authorities);
                 }
+                this.router.navigate(['accessdenied']).then(() => {});
                 return false;
             }
 
             this.stateStorageService.storeUrl(url);
-            this.router.navigate(['accessdenied']).then(() => {
-                // only show the login dialog, if the user hasn't logged in yet
-                if (!account) {
-                    <%_ if (authenticationType !== 'oauth2') { _%>
-                    this.loginModalService.open();
-                    <%_ } else { _%>
-                    this.loginService.login();
-                    <%_ } _%>
-                }
-            });
+            <%_ if (authenticationType !== 'oauth2') { _%>
```

Comment:
Why removing the redirection into the `accessdenied` page?

Reply:
IMO, for authenticated users, we should show access denied page, otherwise, redirect the user to the login page.